saskatoon/saskatoon/urls.py

```diff
@@ -1,18 +1,36 @@
-from django.contrib import admin
-from django.urls import path, include
 import debug_toolbar
 from django.conf import settings
+from django.contrib import admin
+from django.urls import include, path, re_path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Snippets API",
+        default_version='v1',
+        description="Test description",
+        terms_of_service="https://www.google.com/policies/terms/",
+        contact=openapi.Contact(email="contact@snippets.local"),
+        license=openapi.License(name="BSD License"),
+    ),
+    public=True,
+    permission_classes=[permissions.AllowAny],
```

Comment:
@mina-andajos let's at least enforce `IsAuthenticated` permission.
Alternatively we could also write a custom `IsAdmin` permission, similar to `IsCoreOrAdmin` in `harvest.permissons`